File: MODUL-4/SOAL 5/PRAK405-2410817120017-RIKAFAULIANARAHMI.py
n, kelipatan = map(int, input().split())
total = 0

for i in range(1, n + 1):
    baris_total = 0
    for j in range(i, 0, -1):
        baris_total += j * kelipatan
        print(f"({j} * {kelipatan})", end=" ")
        if j > 1:
            print("+ ", end="")
    print(f"= {baris_total}")
    total += baris_total

print(total)

File: MODUL-4/SOAL 5/PRAK405-2410817120017-RIKAFAULIANARAHMI.c
#include <stdio.h>

int main() {
    int n, kelipatan, total = 0;
    scanf("%d %d", &n, &kelipatan);

    for (int i = 1; i <= n; i++) {
        int baris_total = 0;
        for (int j = i; j >= 1; j--) {
            baris_total += j * kelipatan;
            printf("(%d * %d) ", j, kelipatan);
            if (j > 1) printf("+ ");
        }
        printf("= %d\n", baris_total);
        total += baris_total;
    }
    printf("%d\n", total);

    return 0;
}
